model/src/main/kotlin/Scope.kt

```diff
@@ -59,12 +59,12 @@ data class Scope(
     /**
      * Returns whether the given package is contained as a (transitive) dependency in this scope.
      */
-    fun contains(pkg: Package) = contains(pkg.id)
+    operator fun contains(pkg: Package) = contains(pkg.id)
```

Comment:
How is this change related?

Reply:
`operator` is required to be able to use `in` for this function.

Reply:
Interesting. I didn't find this documented anywhere. [This](https://kotlinlang.org/docs/reference/operator-overloading.html#in) just says `a in b` is literally translated to `b.contains(a)`. But I double-checked that the `operator` is indeed required.